src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -1,8 +1,47 @@
 /**
  * A sample SIMID ad that shows a map of nearby locations
  */
+const AdParamKeys = {
+  BUTTON_LABEL: 'buttonLabel',
+  SEARCH_QUERY: 'searchQuery',
+  MARKER: 'marker',
+};
+
+const FIND_NEAREST = "Find Nearest ";
+const DEFAULT_BUTTON_LABEL = "Location";
+
 class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    const adParams = JSON.parse(this.creativeData.adParameters);
+    let buttonLabel = adParams[AdParamKeys.BUTTON_LABEL]; 
```

Comment:
button label should be const

Reply:
right now that I implemented default parameters it can be const